projects/schematics/jest.schematics.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   preset: 'jest-preset-angular',
   setupFilesAfterEnv: ['<rootDir>/test-jest.ts'],
   transform: {
-    '^.+\\.ts?$': 'ts-jest',
+    '^.+\\.(ts|js|html)$': 'jest-preset-angular',
```

Comment:
What is this change about?

Reply:
because `jest-preset-angular` supports Angular 14 https://github.com/thymikee/jest-preset-angular/pull/1611
Using `ts-jest` transformer with Angular 14, will get this error:
![Screen Shot 2022-09-13 at 12 34 29 PM](https://user-images.githubusercontent.com/44440575/189944585-d46d1c75-e93f-4e36-b62d-83327a1b8104.png)